akka-docs/src/main/paradox/typed/dispatchers.md

```diff
@@ -130,50 +125,19 @@ to sleep for an indeterminate time, waiting for an external event to occur.
 Examples are legacy RDBMS drivers or messaging APIs, and the underlying reason
 is typically that (network) I/O occurs under the covers.
 
-Scala
-:   @@snip [BlockingDispatcherSample.scala](/akka-docs/src/test/scala/docs/actor/typed/BlockingDispatcherSample.scala) { #blocking-in-actor }
+### Problem: Blocking on default dispatcher
 
-Java
-:   @@snip [BlockingActor.java](/akka-docs/src/test/java/jdocs/actor/typed/BlockingActor.java)
-
-When facing this, you
-may be tempted to wrap the blocking call inside a `Future` and work
-with that instead, but this strategy is too simple: you are quite likely to
-find bottlenecks or run out of memory or threads when the application runs
-under increased load.
+Simply add blocking calls to your actor message processing like this is problematic:
 
 Scala
-:   @@snip [BlockingDispatcherSample.scala](/akka-docs/src/test/scala/docs/actor/typed/BlockingDispatcherSample.scala) { #blocking-in-future }
+:   @@snip [BlockingDispatcherSample.scala](/akka-docs/src/test/scala/docs/actor/typed/BlockingActor.scala) { #blocking-in-actor }
 
 Java
-:   @@snip [BlockingFutureActor.java](/akka-docs/src/test/java/jdocs/actor/typed/BlockingFutureActor.java) { #blocking-in-future  }
-
-
-### Problem: Blocking on default dispatcher
-
-The key here is this line:
-
-@@@ div { .group-scala }
-
-```scala
-implicit val executionContext: ExecutionContext = context.executionContext
-```
-
-@@@
-
-@@@ div { .group-java }
-
-```java
-ExecutionContext ec = getContext().getExecutionContext();
-```
-
-@@@
-
-Using @scala[`context.executionContext`] @java[`getContext().getExecutionContext()`] as the dispatcher on which the blocking `Future`
-executes can be a problem, since this dispatcher is by default used for all other actor processing
-unless you @ref:[set up a separate dispatcher for the actor](../dispatchers.md#setting-the-dispatcher-for-an-actor).
+:   @@snip [BlockingActor.java](/akka-docs/src/test/java/jdocs/actor/typed/BlockingActor.java)
 
-If all of the available threads are blocked, then all the actors on the same dispatcher will starve for threads and
+Without any further configuration this actor is ran on the default dispatcher along
```

Comment:
"is ran" sounds wrong, past tense, but I'm not sure what would be right, "will run"?